zvariant/src/value.rs

```diff
@@ -230,18 +231,18 @@ impl<'a> Value<'a> {
             Value::Signature(v) => Value::Signature(v.to_owned()),
             Value::ObjectPath(v) => Value::ObjectPath(v.to_owned()),
             Value::Value(v) => {
-                let o = OwnedValue::from(&**v);
+                let o = OwnedValue::try_from(&**v)?;
                 Value::Value(Box::new(o.into_inner()))
             }
 
-            Value::Array(v) => Value::Array(v.to_owned()),
-            Value::Dict(v) => Value::Dict(v.to_owned()),
-            Value::Structure(v) => Value::Structure(v.to_owned()),
+            Value::Array(v) => Value::Array(v.try_to_owned()?),
+            Value::Dict(v) => Value::Dict(v.try_to_owned()?),
```

Comment:
Why were these changed? If `v` implemented to_owned in std, then this should be possible to fail.

Reply:
Those container types have `Value` contained in them so they had to be converted to be fallible as well.